### PR TITLE
v4.8.2

### PR DESCRIPTION
**Pull request by lsm5**
#### Does this PR introduce a user-facing change?

```release-note
None
```

Includes a manual cherrypick of https://github.com/containers/podman/pull/20889 to save CI time.

### CODE REVIEW

**Comment by lsm5** [comment]
@Luap99 @mheon @ashley-cui PTAL

cc @ygalblum @benoitf

File: test/system/252-quadlet.bats [hunk 1]
```diff
@@ -1412,4 +1412,129 @@ EOF
     run_podman rmi --ignore $(pause_image)
 }
 
+@test "quadlet - pod simple" {
```
Comment (ygalblum) [inline test/system/252-quadlet.bats:1415]: `.pod` files are not supported in 4.8.x. This test should not be added

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 4.8.2
+### Bugfixes
+- Fixed a bug in the MacOS pkginstaller where Podman machine was using a different QEMU binary than the one installed using the installer, if it existed on the system ([#20808](https://github.com/containers/podman/issues/20808)).
+- Fixed a bug on Windows (WSL) with the first-time install of user-mode networking when using the init command, as opposed to set ([#20921](https://github.com/containers/podman/issues/20921)).
+
```
Comment (ygalblum) [inline RELEASE_NOTES.md:7]: Should we mention that it also includes the fix for https://github.com/containers/podman/issues/20432?

Comment (lsm5) [reply RELEASE_NOTES.md:7]: sure I can do that

Comment (lsm5) [reply RELEASE_NOTES.md:7]: Done. PTAL

File: test/system/252-quadlet.bats (hunk 1 above)
Comment (lsm5) [reply test/system/252-quadlet.bats]: done. PTAL

**Comment by ygalblum** [comment]
LGTM

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**